cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterHealthService.java

```diff
@@ -54,12 +68,19 @@ public class ClouderaManagerClusterHealthService implements ClusterHealthService
 
     static final String HOST_AGENT_CERTIFICATE_EXPIRY = "HOST_AGENT_CERTIFICATE_EXPIRY";
 
+    // TODO Do we extend this to include additional health check params?
+    static final String NODE_MANAGER_CONNECTIVITY = "NODE_MANAGER_CONNECTIVITY";
+
+    static final String YARN = "yarn";
```

Comment:
Is this the serviceName? Shouldn't this instead be going to the blueprint, figuring out the RoleType=YARN, and pulling the serviceName from the blueprint ?
(Looks like YarnServiceClient in Periscope should do the same as well)

If there's a blueprint where the YARN role is something other than 'yarn' (In the blueprint) for some reason - this will likely break, as well as the check in Periscope.

Reply:
Would this refer to the service refName or the serviceType itself?

Reply:
Changed this to pull the `refName` from the blueprint